adoc/bp_chap_expanded_support.adoc

```diff
@@ -136,6 +136,21 @@ Packages to sync:              3690
 . When the channel has completed the synchronization process you can use the channel as any normal {susemgr} channel.
 
 
+Attempting to synchronize the repository will sometimes fail with this error:
+
+----
+[Errno 256] No more mirrors to try.
+----
+
+To troubleshoot this error, look at the HTTP protocol to determine if [command]``spacewalk-repo-sync`` is running:
+
+.procedure: Debug spacewalk-repo-sync
+. Start debugging mode with [command]``export URLGRABBER_DEBUG=DEBUG``
+. Check the output of [command]``/usr/bin/spacewalk-repo-sync --channel _<channel-label>_ --type yum``
+. If you want to disable debug mode, use [command]``URLGRABBER_DEBUG``
```

Comment:
Huh ? This looks incomplete. Should probably be `unset URLGRABBER_DEBUG` ?!

Reply:
https://github.com/SUSE/doc-susemanager/pull/112